Bricks/Gnuplot/0.0.1.toml

```diff
@@ -0,0 +1,9 @@
+
+[brick]
+name = "Gnuplot"
+version = "0.0.1"
```

Comment:
How about an `authors` field with Marcos Cleison and yourself (as the current package maintainer)?